src/azure-cli/azure/cli/command_modules/storage/operations/azcopy.py

```diff
@@ -70,23 +79,36 @@ def get_url_with_sas(source, account_name, container, blob, share, file_path, lo
                 dir_name = None if dir_name in ('', '.') else dir_name
                 source = client.make_file_url(share, dir_name, file_name)
                 service = 'file'
-            else:  # Only support account transfer for blob
+            else:  # In account level, only blob service is supported
                 source = 'https://{}.blob.core.windows.net'.format(account_name)
                 service = 'blob'
         elif local_path is not None:
             return local_path
         else:
             raise ValueError('Not valid file')
 
-        sas_token = _generate_sas_token(cmd, account_name, account_key, service)
+        # Add sas in url
+        if sas_token:
+            source_sas = sas_token.lstrip('?')
+            return '{}{}{}'.format(source, '?', source_sas)
```

Comment:
sas_token = sas_token.lstrip('?') and remove return '{}{}{}'.format(source, '?', source_sas)

Reply:
done